xx.md

```diff
@@ -0,0 +1,59 @@
+NIP-XX
+======
+
+Video Events
+---------------
+
+`draft` `optional`
+
+This NIP defines several new event kinds representing different types of video media. These events
+are _parameterized replaceable_ as defined in [NIP-01](./01.md) and deletable per [NIP-09](09.md).
+
+Unlike a `kind 1` event with a file attached, video events are intended to be surfaced in an video-specific context
+rather than in a general micro-blogging context.
+
+The following kinds are defined by this NIP:
+
+- `31339` represents a video
+- `31340` represents a short video
+
+The following tags are required:
+
+- `d` is a unique identifier randomly generated by the client.
+- `imeta` is as described in [NIP 92](./92.md). Multiple `imeta` tags MAY be included to provide for different
+  use cases (e.g. streaming vs download).
+- `title` is the title of the video
+
+The following tags are optional:
+
+- `i` is an external GUID in the format defined [here](https://github.com/MerryOscar/nips/commit/280eb498e0ac56b8f9356c1b7a88cc8b31579801).
+- `c` is the video's value as defined by a given category.
+- `website` is an external url to a website related to the video.
```

Comment:
Not sure why you would need this, seems more likely its something you would link inside `content`

Reply:
Again, pulled from audio, based on use cases provided by wavlake and fountain. So it might not transfer over, but could be useful. I can simplify the NIP for the short term.